gradle.properties

```diff
@@ -1,5 +1,5 @@
 group=com.marklogic
-version=1.2.2
+version=1.3.0
```

Comment:
I recommend bumping this up in a separate commit. Makes it easy to look at the commit history and see when the version number was changed. 

Or, change this to 1.3.0-SNAPSHOT to capture that this isn't the real 1.3.0 build yet.

Reply:
"version=1.3.0" makes it match Master. I'll go ahead and bump the version after completing the PR.